gammapy/maps/hpxcube.py

```diff
@@ -369,6 +368,38 @@ def set_by_idx(self, idx, vals):
         idx_local = (self.hpx[idx[0]],) + tuple(idx[1:])
         self.data.T[idx_local] = vals
 
+    def make_cols(self, header, conv):
```

Comment:
This `make_cols` method is part of the public API and visible in the docs, no?
Please either hide it as a private method (add a leading underscore), or add a docstring explaining with one line what it does, and what the inputs / outputs are.

Reply:
You're right.  I've switched this to a private method now.